geomesa-feature/src/main/scala/org/locationtech/geomesa/feature/kryo/SimpleFeatureSerializer.scala

```diff
@@ -237,8 +233,79 @@ object SimpleFeatureSerializer {
           map.entrySet.foreach { e => keyEncoding(out, e.getKey); valueEncoding(out, e.getValue) }
         }
       }
+  }
+
+  /**
+   * Based on the method from geotools WKBWriter. This method is optimized for kryo and simplified from
+   * WKBWriter in the following ways:
+   *
+   * 1. Doesn't save SRID (geomesa didn't use that functionality in WKBWriter)
```

Comment:
This will be fine for now.  

Someday, we'll have to be more accepting of other CRSes;)